src/Pages/ProductDetail/ProductDetail.scss

```
@@ -0,0 +1,254 @@
+.ProductDetail {
+  padding: 0 3rem;
+  color: #484848;
+  font-size: 14px;
+  line-height: 1.5;
+  word-break: keep-all;
+
+  h2 {
+    font-size: 25px;
+  }
+
+  .title {
+    color: #111111;
+    font-weight: 700;
+  }
+
+  .price-wrapper {
+    display: flex;
+    color: black;
+    font-weight: 700;
+    font-size: 22px;
+
+    .won {
+      margin-right: 4px;
+      font-size: 11px;
+    }
+  }
+
+  .round-button {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+  }
+
+  button {
+    background: none;
+    color: #484848;
+    font-size: 14px;
+    line-height: 1.5;
+    border: none;
+    cursor: pointer;
+  }
+
+  img {
+    width: 100%;
+  }
+
+  .modal-button {
+    //mixin
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    //
+    width: 100%;
+    border-top: 1px solid #dfdfdf;
+  }
+
+  .modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba($color: #000000, $alpha: 0.15);
+    z-index: 1000;
+
+    .container {
+      position: absolute;
+      background-color: white;
+
+      .close-button {
+        position: absolute;
+        top: 20px;
+        right: 40px;
+        font-size: 20px;
+
+        &:hover {
+          background-color: #efefef;
+        }
+      }
+    }
+  }
+
+  .detail-main {
+    display: flex;
+    align-items: flex-start;
+    margin-bottom: 3rem;
+
+    .detail-left {
+      width: calc(100% - 340px);
+
+      .left-top {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        margin-bottom: 3rem;
+
+        .image-wrapper {
+          display: flex;
+          align-items: center;
+          margin: 0 10px 20px 10px;
+          //cursor: url(src='../../../public/Image/zoomCursor.png') 20 20, zoom-in;
+        }
+      }
+
+      .left-middle {
+        .summary {
+          margin-bottom: 3rem;
+          font-size: 20px;
+          line-height: 1.6;
+        }
+
+        .information-section {
+          margin-bottom: 3rem;
+
+          button {
+            padding: 2.5rem 0;
+            font-size: 25px;
+
+            span {
+              flex: 1 0 0;
+              text-align: left;
+            }
+
+            &:hover {
+              span {
+                text-decoration: underline;
+              }
+            }
+          }
+        }
+      }
+
+      .left-bottom {
+        h2 {
```

```suggestion
      .left-bottom h2 {
```
이렇게 적어주어도 좋을 것 같고 left-bottom에 부여할 스타일이 없다면 그냥 h2에 직접 클래스명을 주어도 될거 같습니다